src/delivery/git/mod.rs

```diff
@@ -95,13 +95,24 @@ pub fn git_command<P>(args: &[&str], c: &P) -> Result<GitResult, DeliveryError>
         Some(path) => path,
         None => return Err(DeliveryError{ kind: Kind::FailedToExecute, detail: Some("git executable not found".to_owned())}),
     };
-    let mut command = Command::new(command_path);
-    command.args(args);
-    command.current_dir(cwd);
-    debug!("Git command: {:?}", command);
-    let output = match command.output() {
-        Ok(o) => o,
-        Err(e) => { spinner.stop(); return Err(DeliveryError{ kind: Kind::FailedToExecute, detail: Some(format!("failed to execute git: {}", error::Error::description(&e)))}) },
+
+    let output ;
+    if cfg!(target_os = "windows") {
+         output = Command::new(command_path)
```

Comment:
In this part of the if, we're building up the command, configuring it, executing and handling the result in one chain. Below, we are building over several statements. I don't have a strong preference over which way we do it, but it would be nice to be consistent I think.